src/routes/favorites.routes.js

```diff
@@ -12,10 +10,10 @@ const favoritesRoutes = (router) => {
     const response = await findAllFavorites();
     handleResponse(response, res, response.status);
   });
-  router.delete('/favorites/:favoriteId', async (req, res) => {
+
+  router.delete('/favorites/:id', auth, async (req, res) => {
     const { id } = req.params;
-    const response = await getById(id);
-    handleResponse(response, res, response.status);
+    favoriteDeletion(id);
```

Comment:
I'd suggest the name `deleteFavorite` to indicate that this is a proper function.